contracts/PathofTheTemplarShard.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "./PartnerMinter.sol/";
+
+interface IRelic {
+    function balanceOf(address) external returns (uint256);
+    function tokenOfOwnerByIndex(address, uint256) external returns (uint256);
+    function getRelicInfos(uint256 enclaves) external returns (uint256);
+}
+
+interface IShards {
+    function partnerMint(
+        address account,
+        uint256 id,
+        uint256 amount,
+        bytes memory data
+    ) external;
+}
+
+contract PathofTheTemplarShard is ownable {
+
+    event SignedQuestCompletedMessageHash(bytes32 hash);
+
+    IShards private SHARDS,
+    IRelic private RELIC,
+    uint256[] public SHARD_ID = [1, 2, 3, 4, 5];
+
+mapping (msg.sender => bool) public canMint
+mapping (uint => enclaves) signedQuestCompletedMessage
+
+function mintPathofthetemplarShard() external canMint {
```

Comment:
`canMint` here needs to be an actual modifier. Just using a mapping wont work

Reply:
working on figuring out syntax.